diff --git a/dist/index.cjs b/dist/index.cjs
index cd83479144be411bb3b6ee42ea0b40305d9abf37..4146930d9d54388a4df537af5c43e12a3e5dbee7 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -3335,7 +3335,7 @@ function Drawer({
   onOpenChange,
   ...contentProps
 }) {
-  const triggerElement = typeof trigger === "string" ? /* @__PURE__ */ (0, import_jsx_runtime41.jsx)(Button, { ...triggerProps, children: trigger }) : trigger;
+  const triggerElement = typeof trigger === "string" ? /* @__PURE__ */ (0, import_jsx_runtime41.jsx)(Button, { ...triggerProps, children: trigger }) : /* @__PURE__ */ (0, import_jsx_runtime41.jsx)(import_react_aria_components23.Pressable, { children: trigger });
   return /* @__PURE__ */ (0, import_jsx_runtime41.jsxs)(SheetTrigger, { defaultOpen, isOpen, onOpenChange, children: [
     triggerElement,
     /* @__PURE__ */ (0, import_jsx_runtime41.jsx)(SheetContent, { ...contentProps, children })
diff --git a/dist/index.js b/dist/index.js
index 7e00a471dadf0c4cd148ca556d3f5a2f8b16caaf..e3ded913c3dcbdea20d139e39804e981876a0a90 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3083,6 +3083,7 @@ import {
   Modal as ModalPrimitive,
   Dialog as SheetPrimitive,
   DialogTrigger as SheetTriggerPrimitive,
+  Pressable,
   Text as Text3
 } from "react-aria-components";
 import { jsx as jsx41, jsxs as jsxs21 } from "react/jsx-runtime";
@@ -3172,7 +3173,7 @@ function Drawer({
   onOpenChange,
   ...contentProps
 }) {
-  const triggerElement = typeof trigger === "string" ? /* @__PURE__ */ jsx41(Button, { ...triggerProps, children: trigger }) : trigger;
+  const triggerElement = typeof trigger === "string" ? /* @__PURE__ */ jsx41(Button, { ...triggerProps, children: trigger }) : /* @__PURE__ */ jsx41(Pressable, { children: trigger });
   return /* @__PURE__ */ jsxs21(SheetTrigger, { defaultOpen, isOpen, onOpenChange, children: [
     triggerElement,
     /* @__PURE__ */ jsx41(SheetContent, { ...contentProps, children })
